starter-code/css/main.css

```diff
@@ -0,0 +1,73 @@
+*{
+  font-family: Lato, sans-serif;
+}
+body{
+  width: 90%;
+  border-left: solid rgba(173, 173, 173, 1) 5px;
+  padding-left: 40px;
+}
+h1 a{
+  text-decoration: none;
+  color: #ffcb85;
+}
+h1 {
+  font-weight: 300;
+  font-size: 48px;
+}
+h2 {
+  font-weight: 400;
+  margin-top: -1em
+}
+nav a {
+  color: rgba(227, 227, 227, 1);
+  font-size: 20px;
+  padding: 0 6px;
+}
+h3 {
+  color: rgba(227, 227, 227, 1);
+  font-size: 35px;
+  font-weight: lighter;
+  padding-top: 2%;
+}
+h2 a{
+  text-decoration: none;
+  color: black;
+  font-weight: 300;
+  font-size: 30px;
+}
+img{
+  width: 630px;
+}
+p::first-letter{
+  color: rgba(227, 227, 227, 1);
+  font-size: 90px;
+  line-height: 70%;
+  float: left;
+  padding: 0 8px 8px 0;
+}
+p{
+  font-size: 17px;
+}
+h4 a{
+  text-decoration: none;
+  color: #ffcb85;
+  text-align: right;
+  padding-left: 80%;
+}
+article{
+  width: 59%;
+  border-bottom: solid rgba(227, 227, 227, 1) 2px;
+}
+footer{
+  padding-top: 7%;
+  color: rgba(227, 227, 227, 1);
+
+}
+footer nav a{
+  color: #ffcb85;
```

Comment:
Sometimes you are setting the color with RGB, other times it's with hex code. Try to be consistent in what you choose so it's easier to identify colors that are the same.